tools/README.md

```diff
@@ -29,6 +29,8 @@ Build system pieces:
 * [build_libtorch.py](build_libtorch.py) - Script for building
   libtorch, a standalone C++ library without Python support.  This
   build script is tested in CI.
+* [fast_nvcc](fast_nvcc) - Mostly-transparent wrapper script that parallelizes
```

Comment:
i believe it follows markdown syntax (e.g. `[link_display_name](link_relative_path)`
```suggestion
* [fast_nvcc.py](fast_nvcc/fast_nvcc.py) - Mostly-transparent wrapper script that parallelizes
```

Reply:
I'd rather leave it as-is to match the existing convention in this README (e.g. for `docker/Dockerfile_runtime`). If you want, though, I could change it to a bullet with a subbullet, similar to what's currently done for `amd_build/build_amd.py` and `shared/module_loader.py`. Preference?

Reply:
oh. I see. are you planning to add more to the folder `fast_nvcc`? if yes, then the current bullet `[fast_nvcc]` looks good! and yes if you can let's add a subbullet for `fast_nvcc/fast_nvcc.py`.

Reply:
I don't currently have any plans to add more files to that folder, but I needed it to be in its own folder for hg dirsync to work; but yea I'll add a subbullet :)